config/scenario_config.csv

```diff
@@ -4,7 +4,7 @@ SSP2-NPi-calibrate;calibrate,AMT,compileInTests,calibrateSSP2;calibrate;14;;;;;;
 SSP2-NDC;1,AMT,2;;;;;;globallyOptimal;;;3;;NDC;;;;;;;;;;;;;;;;;;NDC;;2030.EUR_regi.tax.t.FE_wo_b_wo_n_e.all 1.2809;2020.2030.EUR_regi.all.year.netGHG_LULUCFGrassi 2.450;;;;;;;;;;;;;;;;;;;;1;2030;;SSP2-NPi2025;SSP2-NPi;SSP2-NPi2025;SSP2-NDC: This Nationally Determined Contribution (NDC) scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road. It assumes NPi2025 until 2025 and has start year 2030. The NDC includes all pledged policies even if not yet implemented. It assumes that the moderate and heterogeneous climate ambition reflected in the NDCs at the begining of 2021 continues over the 21st century.
 SSP2-NDCsy2025;1,AMT,2;;;;;;globallyOptimal;;;3;;NDC;;;;;;;;;;;;;;;;;;NDC;;2030.EUR_regi.tax.t.FE_wo_b_wo_n_e.all 1.2809;2020.2030.EUR_regi.all.year.netGHG_LULUCFGrassi 2.450;;;;;;;;;;;;;;;;;;;;5;2025;;SSP2-NPi;SSP2-NPi;SSP2-NPi2025;SSP2-NDCsy2025: This Nationally Determined Contribution (NDC) scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road. It assumes NPi until 2020 and has start year 2025.The NDC includes all pledged policies even if not yet implemented. It assumes that the moderate and heterogeneous climate ambition reflected in the NDCs at the begining of 2021 continues over the 21st century.
 SSP2-NPi;1,AMT,compileInTests,2;;;;;;;;;0;;NPi;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;Mix1;;2005;;;;;SSP2-NPi: This National Policies Implemented (NPi) scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road. The NPi assumes that policies fail to achieve NDC targets in 2030. Instead, carbon prices are assumed to grow and converge more slowly, leading to emissions trajectories in line with bottom-up studies on the effect of currently implemented policies.
-SSP2-NPi2025;1,AMT,2;;;;;;;;;0;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;2005;;;;;SSP2-NPi2025: This National Policies Implemented (NPi) scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road. The NPi assumes that policies fail to achieve NDC targets in 2030. Instead, carbon prices are assumed to grow and converge more slowly, leading to emissions trajectories in line with bottom-up studies on the effect of currently implemented policies.
+SSP2-NPi2025;1,AMT,2;;;;;;;;;0;;;;;;;;;;;;;;;;;;;;NPi2025;;;;;;;;;;;;;;;;;;;;;;;;2005;;;;;SSP2-NPi2025: This National Policies Implemented (NPi) scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road. The NPi assumes that policies fail to achieve NDC targets in 2030. Instead, carbon prices are assumed to grow and converge more slowly, leading to emissions trajectories in line with bottom-up studies on the effect of currently implemented policies.
```

Comment:
There seems to be something wrong in this line. A copy of the `SSP2-PkBudg500`-line seems to be attached at the end.

Reply:
If you change the defaults for SSP2-NPi2025, please do so in the `main.gms` and leave the columns in the `scenario_config.csv` empty. This is, because we want the `main.gms` to reflect the SSP2-NPi, so that, if you run the model without changing any switches in main.gms you get a SSP2-NPi2025. Thank you!